types/react-bootstrap/lib/ToggleButtonGroup.d.ts

```diff
@@ -3,37 +3,36 @@ import { ButtonGroup } from 'react-bootstrap';
 import { Omit } from "../index";
 
 declare namespace ToggleButtonGroup {
-    interface BaseProps {
```

Comment:
In general I'm in favor of respecting the project's existing indentation style which seems to be 4 spaces. There isn't a lint rule one way or another so I wouldn't consider this a blocker, but someone might.